content/widgets/answers.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Answers
+type: widget
+html: |
+  <div class="ais-Answers">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list">
+      <li class="ais-Answers-item">Author Sam Martin shares photos of ...</li>
+      <li class="ais-Answers-item">Beau Lotto's color games puzzle your vision, but they also spotlight what you can't normally see...</li>
+    </ul>
+  </div>
+alt1: empty
+althtml1: |
+  <div class="ais-Answers ais-Answers--empty">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list"></ul>
+  </div>
+alt2: loading
+althtml2: |
+  <div class="ais-Answers ais-Answers--empty">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <div class="ais-Answers-loader">loading...</div>
+    <ul class="ais-Answers-list"></ul>
```

Comment:
when loading, there shouldn't be a list (empty or not) I think

Reply:
https://github.com/algolia/instantsearch-specs/pull/113/commits/e8a3d3f237c670a76ac99f8227ef9cebe9726872